applications/composer/frontend/package.json

```diff
@@ -88,6 +88,7 @@
     "@typescript-eslint/parser": "^6.14.0",
     "eslint": "^8.55.0",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-react-refresh": "^0.4.5"
+    "eslint-plugin-react-refresh": "^0.4.5",
+    "typescript": "^5.8.3"
```

Comment:
@Salam-Dalloul the build for this is failing due to the package-lock not being updated with the new dependency used, could you please fix this? Thanks!